test/intl402/NumberFormat/test-option-useGrouping-extended.js

```diff
@@ -35,6 +35,8 @@ assert.sameValue(render({notation: 'compact', useGrouping: false}), false, 'comp
 assert.sameValue(render({notation: 'compact', useGrouping: null}), false, 'compact, null');
 assert.sameValue(render({notation: 'compact', useGrouping: 'min2'}), 'min2', 'compact, "min2"');
 
-assert.sameValue(render({useGrouping: 'undefined'}), 'auto', 'use fallback value');
+assert.throws(RangeError, function () {
```

Comment:
Thanks for catching this @FrankYFTang  I would rather suggest deleting this assertion and add "undefined" string at https://github.com/tc39/test262/blob/main/test/intl402/NumberFormat/test-option-useGrouping.js#L34